packages/flutter_tools/lib/src/ios/xcresult.dart

```diff
@@ -236,3 +260,39 @@ enum XCResultIssueType {
   /// This is for all the issues under the `errorSummaries` key in the xcresult.
   error,
 }
+
+// A typical location url string looks like file:///foo.swift#CharacterRangeLen=0&EndingColumnNumber=82&EndingLineNumber=7&StartingColumnNumber=82&StartingLineNumber=7.
+//
+// This function converts it to something like: file:///foo.swift:<StartingLineNumber>:<StartingColumnNumber>.
+String _convertUrlToLocationString(String url){
+  if (url.isEmpty) {
+    return '';
+  }
+  final List<String> topComponents = url.split('#');
+  final String filePath = topComponents.first;
+  if (topComponents.length == 1) {
+    return filePath;
```

Comment:
This probably won't happen. Since it is not a json that we maintain, I'm not sure all the possibilities that could be. So just in case there are no parameters, we can show the file path at least.